tags/java.md

```diff
@@ -1 +1,8 @@
-https://prismlauncher.org/wiki/getting-started/installing-java/
+---
+embed:
+  title: Java Instructions 
+  color: orange
+---
+
+For security reasons, Prism Launcher does not bundle in Java with itself. The instructions to setup and install Java can be found [here](https://prismlauncher.org/wiki/getting-started/installing-java/).
+***DO NOT USE ORACLE!!!***
```

Comment:
yes the webpage also states this, but is it really worth having this here?
instead it might be better to just directly poste the whole faq post from the webpage, but that's too much...

(non issue here) also the webpage post isn't the best for quickly figuring out where to get java...

Reply:
I can rework the webpage when I get home from school. I removed the Oracle bit since I've come to the conclusion that it ultimately doesn't matter.